install/scripts/aws-up.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -e
+set -u
+
+# Required env variables:
+: "$AWS_ACCOUNT_ID $AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY"
+
+INSTALL_OPERATOR="${INSTALL_OPERATOR:-yes}"
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+KUBERENTES_DIR=${SCRIPT_DIR}/../kubernetes
+
+EKSCTL_ENABLE_CREDENTIAL_CACHE=1
+export CLUSTER_NAME=substratus
```

Comment:
Preference for following bash conventions of using lowercase variables within scripts:

```bash
cluster_name=substratus
```

https://stackoverflow.com/questions/673055/correct-bash-and-shell-script-variable-capitalization

Exception would be the variables that you are using `envsubst` with.

Reply:
done.